Packs/ApiModules/Scripts/TAXII2ApiModule/TAXII2ApiModule.py

```diff
@@ -919,11 +924,58 @@ def change_attack_pattern_to_stix_attack_pattern(indicator: dict[str, Any]):
 
         return indicator
 
+    @staticmethod
+    def get_entity_b_type_and_value(related_obj: str, id_to_object: dict[str, dict[str, Any]],
+                                    is_unit42_report: bool = False) -> tuple:
+        """
+        Gets the type and the value of the indicator.
+
+        Args:
+            related_obj: the indicator to get information on.
+            id_to_object: a dict in the form of - id: stix_object.
+            is_unit42_report: represents whether unit42 report or not.
+
+        Returns:
+            tuple. the indicator type and value.
+        """
+        indicator_obj = id_to_object.get(related_obj, {})
+        entity_b_value = indicator_obj.get('name', '')
+        entity_b_obj_type = STIX_2_TYPES_TO_CORTEX_TYPES.get(indicator_obj.get('type', ''),
+                                                             STIX2XSOARParser.get_ioc_type(related_obj, id_to_object))
+        if indicator_obj.get('type') == "indicator":
+            entity_b_value = STIX2XSOARParser.get_ioc_value(related_obj, id_to_object)
+        elif indicator_obj.get('type') == "attack-pattern" and is_unit42_report:
+            _, entity_b_value = STIX2XSOARParser.get_attack_id_and_value_from_name(indicator_obj)
+        elif indicator_obj.get('type') == "report" and is_unit42_report:
+            entity_b_value = f"[Unit42 ATOM] {indicator_obj.get('name')}"
+        return entity_b_obj_type, entity_b_value
+
+    @staticmethod
+    def get_attack_id_and_value_from_name(attack_indicator):
+        """
+        Split indicator name into MITRE ID and indicator value: 'T1108: Redundant Access' -> MITRE ID = T1108,
+        indicator value = 'Redundant Access'.
+        """
+        ind_name = attack_indicator.get('name')
+        separator = ':'
```

Comment:
```suggestion
        separator = ': '
```
If the template of '<mitre-id>: <value>' is constant, we can make the seperator = ': ' to simplify the rest of the function

Reply:
I didn't write this function just copy from the unit42 code. 
Don't think I need to change legacy code.